library/Isochrone_IGN_API.py

```diff
@@ -49,7 +52,7 @@ def __init__(self, input_layer:gpd.GeoDataFrame, range_value:list[int], processi
 
     @staticmethod
     @decorators.retryRequest(min_wait=1, wait_multiplier=2, max_retries=3)
-    def request_IGN_isochrone_api(point:str , costValue:int, resource:str='bdtopo-valhalla', costType:str="time", profile:str='car', direction:str='arrival', constraints:str=None, geometryFormat:str='geojson', distanceUnit:str='meter', timeUnit:str='minute', crs:str='EPSG:4326') -> gpd.GeoDataFrame:
+    def request_IGN_isochrone_api(point:str , costValue:int,valueKey:str=None, resource:str='bdtopo-valhalla', costType:str="time", profile:str='car', direction:str='arrival', constraints:str=None, geometryFormat:str='geojson', distanceUnit:str='meter', timeUnit:str='minute', crs:str='EPSG:4326') -> gpd.GeoDataFrame:
```

Comment:
```suggestion
    def request_IGN_isochrone_api(point:str , costValue:int, resource:str='bdtopo-valhalla', costType:str="time", profile:str='car', direction:str='arrival', constraints:str=None, geometryFormat:str='geojson', distanceUnit:str='meter', timeUnit:str='minute', crs:str='EPSG:4326') -> gpd.GeoDataFrame:
```

I'm not adding valueKey here because I kinda want to keep this function as close as possible to an API request. I don't want to add parameters that could create confusion if used into another tool !
Instead I will add valueKey in the main() function just after using request_IGN_isochrone_api() so that the gdf returned as output contain the attribute.